sys/shell/shell.c

```diff
@@ -256,6 +256,9 @@ static int readline(char *buf, size_t size)
         else {
             *line_buf_ptr++ = c;
             _putchar(c);
+#ifdef MODULE_NEWLIB
+            fflush(stdout);
+#endif
```

Comment:
> about shell you are right, formally it is not related to this cpu, the important thing is that if it is a bug then it has to be pulled into.

Can you extract this into a separate PR then, please? I'm not so sure, if want to consider this as an actual "bug", because the shell is based on linebreak, so flushing every single character seems to bring little benefit.